app/models/user.rb

```diff
@@ -27,6 +25,6 @@ def can_delete?
   end
 
   def to_s
-    "#{name} <#{email}>"
+    email
```

Comment:
This results in a change when displaying audits that will be visible to users and looks less clean, but I think it's worth it.